docs/docs/index.md

```diff
@@ -4,31 +4,32 @@ description: The one stop location for tutorials, guides, and information about
 disableTableOfContents: true
 ---
 
-Gatsby is a blazing fast modern site generator for React.
+Gatsby React tabanlı oldukça hızlı bir modern site oluşturucusudur.
 
-## Get Started
+## Başlarken
 
-There are two main ways to get started with Gatsby:
+Gatsby'ye başlamanın 2 yolu var:
 
-1. [Tutorial](/tutorial/): Step-by-step instructions on how to install Gatsby and start a project: written for people without Gatsby or web development experience, though these learning resources have helped developers of all skill levels.
-2. [Quick start](/docs/quick-start): One page summary of how to install Gatsby and start a new project.
+1. [Eğitim](/tutorial/): Adım adım talimatları izleyerek sıfırdan Gatsby kurulumu yaparak bir proje oluşturun: Daha önce hiç Gatsby ya da web geliştirme deneyimi olmayan, ancak her beceri seviyesindeki yazılımcılara da yardımcı olur.
 
-## Go further
+2. [Hızlı Başlangıç](/docs/quick-start): Gatsby kurulumunun ve bir proje oluşturmanın tek sayfada özetlenmiş halidir.
 
-1. [Recipes](/docs/recipes/): Find some quick answers for how to accomplish some specific, common tasks with Gatsby.
-2. Choose your own adventure and peruse the various sections of the Gatsby docs:
+## Daha İlerisi
 
-   - [Reference Guides](/docs/guides/): Learn about the many different topics around building with Gatsby, like sourcing data, deployment, and more.
-   - [Gatsby API Reference](/docs/api-reference/): Learn more about Gatsby APIs and configuration.
-   - [Releases & Migration](/docs/releases-and-migration/): Find release notes and guides for migration between major versions.
-   - [Conceptual Guide](/docs/conceptual-guide/): Read high-level overviews of the Gatsby approach.
-   - [Gatsby Internals](/docs/gatsby-internals/): Dig into how Gatsby works behind the scenes.
-   - [Using Gatsby Professionally](/docs/using-gatsby-professionally/): Learn tips and tricks for how to explain Gatsby to others at work, so that you have more opportunities to work with Gatsby professionally.
+1. [Tarifler](/docs/recipes/): Gatsby'deki belirli konulardaki ortak sorunların çözümlerine ilişkin hızlı yanıtları bulun.
```

Comment:
Can you change `Gatsby'de, belirli konulardaki ortak sorunların ...` instead of `Gatsby'deki belirli konulardaki ortak sorunların ...`? I think this feels more right.